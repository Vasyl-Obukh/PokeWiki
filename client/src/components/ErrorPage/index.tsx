import * as React from 'react';
import styled from 'styled-components';

const ErrorMessage = styled.h1`
  margin-top: 100px;
  color: red;
  text-align: center;
`;

const ErrorPage = () => <ErrorMessage>Error Page</ErrorMessage>;

export default ErrorPage;