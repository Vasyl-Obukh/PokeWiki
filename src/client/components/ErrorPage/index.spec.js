import React from 'react';
import renderer from 'react-test-renderer';
import ErrorPage from './index';

describe('<ErrorPage /> tests', () => {
  test('snapshot test', () => {
    const component = renderer.create(<ErrorPage/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});