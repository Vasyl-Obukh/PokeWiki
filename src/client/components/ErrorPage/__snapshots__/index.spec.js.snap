// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> tests snapshot test 1`] = `
<h1
  className="sc-bdVaJa grGzbq"
>
  Error Page
</h1>
`;
